src/bin/common/parse_command.ml

```diff
@@ -0,0 +1,1157 @@
+         $(docv) must be %s. It must be noticed that not specifying an output \
+         format will let Alt-Ergo set it according to the input file's \
+         extension."
+        (Arg.doc_alts [ "native"; "smtlib" ])
+    in
+    let docv = "FMT" in
+    Arg.(value & opt (some format_conv) None & info ["o"; "output"] ~docv ~doc)
+  in
+
+  Term.(ret (const mk_output_opt $
+             interpretation $ model $ unsat_core $ output_format
+            ))
+
+let parse_profiling_opt =
+
+  let docs = s_profiling in
+
+  let cumulative_time_profiling =
+    let doc =
+      "Record the time spent in called functions in callers" in
+    Arg.(value & flag & info ["cumulative-time-profiling"] ~docs ~doc) in
+
+  let profiling =
+    let doc =
+      "Activate the profiling module with the given frequency. \
+       Use Ctrl-C to switch between different views and \\\"Ctrl \
+       + AltGr + \" to exit." in
+    let docv = "DELAY" in
+    Arg.(value & opt (some float) None & info ["profiling"] ~docv ~docs ~doc) in
+
+  let profiling_plugin =
+    let doc = "Use the given profiling plugin." in
+    let docv = "PGN" in
+    Arg.(value & opt string (get_profiling_plugin ()) &
+         info ["profiling-plugin"] ~docv ~docs ~doc) in
+
+  let get_verbose =
+    let doc = "Set the get_verbose mode." in
+    Arg.(value & flag & info ["v"; "verbose"] ~doc) in
+
+  Term.(ret (const mk_profiling_opt $
+             cumulative_time_profiling $ profiling $
+             profiling_plugin $ get_verbose
+            ))
+
+let parse_quantifiers_opt =
+
+  let docs = s_quantifiers in
+
+  let greedy =
+    let doc = "Use all available ground terms in instantiation." in
+    Arg.(value & flag & info ["g"; "greedy"] ~doc) in
+
+  let instantiate_after_backjump =
+    let doc =
+      "Make a (normal) instantiation round after every backjump/backtrack." in
+    Arg.(value & flag & info ["inst-after-bj"] ~docs ~doc) in
+
+  let max_multi_triggers_size =
+    let doc = "Max number of terms allowed in multi-triggers." in
+    let docv = "VAL" in
+    Arg.(value & opt int (get_max_multi_triggers_size ()) &
+         info ["max-multi-triggers-size"] ~docv ~docs ~doc) in
+
+  let nb_triggers =
+    let doc = "Number of (multi)triggers." in
+    let docv = "VAL" in
+    Arg.(value & opt int (get_nb_triggers ()) &
+         info ["nb-triggers"] ~docv ~docs ~doc) in
+
+  let no_ematching =
+    let doc = "Disable matching modulo ground equalities." in
+    Arg.(value & flag & info ["no-ematching"] ~docs ~doc) in
+
+  let no_user_triggers =
+    let doc = "Ignore user triggers, except for triggers of theories axioms"; in
+    Arg.(value & flag & info ["no-user-triggers"] ~docs ~doc) in
+
+  let normalize_instances =
+    let doc =
+      "Normalize generated substitutions by matching w.r.t. the state of \
+       the theory. This means that only terms that \
+       are greater (w.r.t. depth) than the initial terms of the problem are \
+       normalized." in
```

Comment:
I don't understand what this means. What is the normalization process ? Does the normalization process applies to the substitution or the instance ?

Reply:
I don't know.